src/vs/workbench/contrib/debug/browser/breakpointsView.ts

```diff
@@ -1472,7 +1616,7 @@ registerAction2(class extends ViewAction<BreakpointsView> {
 				id: MenuId.DebugBreakpointsContext,
 				group: 'navigation',
 				order: 20,
-				when: CONTEXT_BREAKPOINT_ITEM_TYPE.isEqualTo('functionBreakpoint')
+				when: ContextKeyExpr.or(CONTEXT_BREAKPOINT_ITEM_TYPE.isEqualTo('functionBreakpoint'), CONTEXT_BREAKPOINT_ITEM_TYPE.isEqualTo('dataBreakpoint'))
```

Comment:
I noticed when looking at the precondition that this is enabled when `supportsConditionalBreakpoints`, not when `supportsHitConditionalBreakpoints`. Not an issue in this PR, but something we should fix 🤔